packages/salesforcedx-vscode-core/src/commands/forceApexLogGet.ts

```diff
@@ -63,13 +63,18 @@ export class ForceApexLogGetExecutor extends SfdxCommandletExecutor<
   public async execute(
     response: ContinueResponse<ApexDebugLogIdStartTime>
   ): Promise<void> {
+    const startTime = process.hrtime();
     const cancellationTokenSource = new vscode.CancellationTokenSource();
     const cancellationToken = cancellationTokenSource.token;
     const execution = new CliCommandExecutor(this.build(response.data), {
       cwd: vscode.workspace.rootPath
     }).execute(cancellationToken);
     this.attachExecution(execution, cancellationTokenSource, cancellationToken);
-    this.logMetric(execution.command.logName);
+
+    execution.processExitSubject.subscribe(async data => {
```

Comment:
@lcampos I don't think it will break the functionality, but the 'async' keyword isn't needed here and is used in several other places where it isn't necessary. 

Eslint has a rule 'require-await' that flags the usage of 'async' without 'await', but tslint does not have the same rule.

Reply:
Good catch, let me update all the classes that have a similar pattern. Might end up even removing data, if possible, since I'm not using it.